managed/src/main/java/com/yugabyte/yw/models/InstanceType.java

```diff
@@ -210,6 +210,7 @@ public static InstanceType createWithMetadata(Provider provider, String instance
 
     public List<VolumeDetails> volumeDetailsList;
     public PublicCloudConstants.Tenancy tenancy;
+    public Integer volumeCount= DEFAULT_VOLUME_COUNT;
```

Comment:
Why not put the aws specific change in this class in  `setVolumeDetailsList`  method?
add `setvolumeCount()` after `setDefaultMountPaths()` and in that method do aws specific hack. 
Also if you can make it `logical` instead of just whitelisting 'c5.' 'c4.' prefix that would be even better. 
That would actually be a permanent fix.